rpcs3/Gui/MainFrame.cpp

```diff
@@ -73,7 +73,7 @@ MainFrame::MainFrame()
 	, m_sys_menu_opened(false)
 {
 
-	SetLabel(wxString::Format(_PRGNAME_ " " _PRGVER_ " " RPCS3_GIT_VERSION));
+	SetLabel(wxString::Format(_PRGNAME_ " " RPCS3_GIT_VERSION));

```

Comment:
@Nekotekina If, in fact, I have fixed the problem, and that was the case before, it was you who did it again at once, whereas it was like that for a while now, since I on the works again git rpcs3
you can not remove the tag while keeping the number (on your Example 13) is the only solution, and any other background emulator such as me, so where is the problem.